PlayerControls/sources/AdVideoControls.swift

```diff
@@ -75,7 +75,7 @@ public final class AdVideoControls: UIViewController {
         public let isLoading: Bool
         public let airplayActiveViewHidden: Bool
         
-        public enum MainAction {
+        public enum MainAction: GeneratePrism {
```

Comment:
Maybe just `Prism`?

@AlexeyDemedetskiy Your thoughts?

Reply:
In my opinion, `GeneratePrism` means "generate prism for this Enum", but `Prism` doesn't sound so evident

Reply:
Nobody names protocols like this. `Prism` is pretty self explanatory. And this is empty protocol - so the only purpose of it - marking.